.github/workflows/Operator-CI.yaml

```diff
@@ -0,0 +1,69 @@
+name: Shard Operator - CI
+on:
+  pull_request:
+    paths:
+      - 'shard-operator/**'
+jobs:
+  event-bridge-build:
+    env:
+      MAVEN_OPTS: "-Dhttps.protocols=TLSv1.2 -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=WARN -Dorg.slf4j.simpleLogger.showDateTime=true -Djava.awt.headless=true -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120"
+      NAMESPACE: mynamespace
+    concurrency:
+      group: event-bridge-operator-pr-${{ github.head_ref }}
+      cancel-in-progress: true
+    timeout-minutes: 45
+    runs-on: ubuntu-latest
+    name: Build and Test Operator
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache Maven Packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+      - name: Set up KinD
+        uses: container-tools/kind-action@v1
+        with:
+          config: .github/kind-config.yaml
+          registry: false                  # Registry not needed
+          node_image: kindest/node:v1.22.2 # Newest image
+      - name: Check cluster status
+        run: |
+          kubectl get nodes
+          kubectl cluster-info
+      - name: Install Ingres
+        run: |
+          kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+          kubectl wait --namespace ingress-nginx --for=condition=ready pod --selector=app.kubernetes.io/component=controller --timeout=90s
+        # Tests are executed in different PR check, we just need all artifacts in repository
+      - name: Build complete repo
+        shell: bash
+        run: mvn clean install --no-snapshot-updates --fail-at-end --errors --quiet -DskipTests
+        # Build operator image and generate needed operator resources
+      - name: Build Shard operator image and resources
+        shell: bash
+        working-directory: shard-operator
+        run: mvn clean install --errors --quiet -Pminikube -Devent-bridge.k8s.platform=k8s -Dnamespace=$NAMESPACE
```

Comment:
If I'm not wrong once https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/113 is merged you don't need anymore to specify `-Devent-bridge.k8s.platform=k8s`. But please double check :)

Reply:
I think so, will try once it is merged.